lib/tests-from-pull-request.js

```diff
@@ -1,32 +1,38 @@
-const fs = require('fs')
-const readline = require('readline')
+module.exports = getTestsFromPR;
 
-module.exports = getTestsFromPR
-
-function getTestsFromPR (context) {
+function getTestsFromPR(context) {
   return Promise.resolve()
-    .then(() => ['Hi Or'])
-  // .then(() => _getTestsFromPR(context))
+    .then(() => _getTestsFromPR(context));
 }
 
-function _getTestsFromPR (context) {
-  const pull = context.issue()
-  const {repos, pullRequests} = context.github
-  // todo: code to get file contect
-  let newTestLines = []
-  let PRfiles
+function _getTestsFromPR(context) {
+  let PRfiles;
+  const pull = context.issue();
+  const {pullRequests} = context.github;
+
   return pullRequests.getFiles(pull)
     .then((res) => res.data)
     .then((files) => PRfiles = files)
-    .then(() => console.log('FILES FROM PR:\n', PRfiles))
+    .then(() => PRfiles.map((file) => file.patch))
+    .then((patches) => patches.join('\n'))
+    .then(_getTestNamesFromPatch);
+}
+
 
-  // let line = ''
-  // if (line.startsWith('+')) {
-  //   line = line.slice(1).trim() // remove the + and spaces
-  //   if (line.trim().startsWith('it(')) {
-  //     newTestLines.push(line)
-  //   }
-  // }
+function _getTestNamesFromPatch(patch) {
```

Comment:
I would call it `prPatch` or something (the func and variable names) just to distinguish between a file patch and an entire pull request patch